electron-app/src/store/balances/getters.ts

```diff
@@ -121,13 +122,29 @@ export const getters: GetterTree<BalanceState, RotkehlchenState> = {
     if (!ethAccount || isEmpty(ethAccount)) {
```

Comment:
FYI: I can't see the `accountTokens` getter used anywhere else in the code. It seems that it has been replaced with `accountAssets`. Doing a quick search returns only the declaration. Can you also verify that there is no usage? If it is not used then we can remove it.

Reply:
indeed